src/syntax/commentsLexer.mll

```diff
@@ -102,6 +102,8 @@ and comment n ignored marked stack = parse
                                 normal ignored marked stack lexbuf
                               else
                                 comment (pred n) ignored marked stack lexbuf }
+| "'\"'"                    { comment n ignored marked stack lexbuf }
+| "'\\\"'"                  { comment n ignored marked stack lexbuf }
```

Comment:
Similar to the string case, if we're lexing a comment, the default pattern on line 110 seems to catch these.